engine/tox.ini

```diff
@@ -9,6 +9,6 @@ banned-modules =
 [pytest]
 # https://pytest-django.readthedocs.io/en/latest/configuring_django.html#order-of-choosing-settings
 # https://pytest-django.readthedocs.io/en/latest/database.html
-addopts = --color=yes --showlocals
+addopts = --no-migrations --color=yes --showlocals
```

Comment:
This is needed to allow creating models on demand in tests, e.g. `TestOrderedModel`.
Without `--no-migrations` it would be required to store migration for such test models.